robottelo/ui/base.py

```diff
@@ -526,7 +526,8 @@ def click(self, locator, wait_for_ajax=True,
               ajax_timeout=30, waiter_timeout=12, scroll=True):
         """Locate the element described by the ``locator`` and click on it.
 
-        :param locator: The locator that describes the element.
+        :param locator: Could be either locator that describes the element or
```

Comment:
I think specifying accepted types for locator parameter would be appropriate (tuple | WebElement).